plugins/techdocs/api-report.md

```diff
@@ -23,115 +23,51 @@ import { TableColumn } from '@backstage/core-components';
 import { TableProps } from '@backstage/core-components';
 import { UserListFilterKind } from '@backstage/plugin-catalog-react';
 
-// Warning: (ae-missing-release-tag) "createCopyDocsUrlAction" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
-function createCopyDocsUrlAction(copyToClipboard: Function): (
```

Comment:
Want to ping @kuangp on possibly removing some of the below from the API interface of the plugin -- I'm not sure how these are actually used in practice.

Reply:
yes related to that I'm not sure about https://github.com/backstage/backstage/pull/9730/files#diff-0993a6b9ee4d31d7bc0e3837ee21571bb4510dbe621ef3b66e76aa4de8267868R368. Would it be enough exporting the factories? @kuangp?

Reply:
made some additional changes to this, see https://github.com/backstage/backstage/pull/9730/commits/6d55364efdadb03fe73b3dc399739ae9357cf490 and let me know what you think.

Reply:
Technically these are breaking (`minor` level changeset), I guess.

Reply:
Exporting these through factories on the tables is fine with us! That's actually how we're importing already :) 

For our use case, we're expanding the default set of actions on the TechDocs table and so are reusing the existing ones.

Reply:
Great, will add a minor changeset and instructions for use then! Thanks for letting us know!